functions/go/apply-setters/applysetters/apply_setters.go

```diff
@@ -202,10 +202,14 @@ func currentSetterValues(pattern, value string) map[string]string {
 	res := make(map[string]string)
 	// get all setter names enclosed in ${}
 	urs := unresolvedSetters(pattern)
+	// escape pattern
```

Comment:
Add an example pattern as a comment which needs this. `my-app-layer.${stage}.${domain}.${tld}`

Reply:
Actually, you can update entire example `foo-${image-setter}:${tag-setter}-bar` with `my-app-layer.${stage}.${domain}.${tld}` and add transformations as comments for each variable.

Reply:
I've replaced the example with the sugested pattern.